### DISCUSSION TITLE
WebParts PNP Search Results HTML Fields

### DISCUSSION THREAD

**Discussion by bucrepus** [microsoft-search/pnp-modern-search#2855] [Q&A] [answered]
The documentation under MANAGE CARD FIELDS states "Also, if the field doesn't have the 'Allow HTML' indication flag enabled, it means the value will be always interpreted as text, regardless if you set an HTML value. Otherwise, your value will be interpreted as HTML for those fields (ex: 'Location' and 'Tags' placeholder fields). "
Where can you make the other fields HTML enabled? The example stated above does show Location and Tags as being HTML enabled (and they are in the MANAGE CARD FIELDS dialog), but how does one turn some of the other fields on? It looks like the example fields are actually hardcoded to be HTML and those are the only ones that can be used. Any ideas where to turn on the 'flag' on other fields in the default list?
Thanks for your time..

![manage_cards_fields](https://user-images.githubusercontent.com/17578967/225418711-3ea2336d-4e95-401d-b134-22f2438a4337.png)

**Comment by wobba** [comment, marked as answer]
This refers to the two card slots Location and Tags allowing HTML. The other card slots don't allow for rendering arbitrary html from a component point of view. Which is why they are called out and highlighted.

**Reply by bucrepus** [reply]
Thanks. They have EX: in front of them so I assumed they were just example fields being listed. Thanks for your time,
